entry.py

```diff
@@ -0,0 +1,9 @@
+from aiohttp.web import run_app
+
+from api_demo.app import create_app
+
+if __name__ == "__main__":
+    app = create_app()
+    run_app(
+        app,
+    )
```

Comment:
Add also host and port from config

Reply:
What do you mean, when do you want to add host and port from config?